addons/mail/static/src/models/attachment_list/attachment_list.js

```diff
@@ -6,7 +6,7 @@ import { clear, insertAndReplace, replace } from '@mail/model/model_field_comman
 
 registerModel({
     name: 'AttachmentList',
-    identifyingFields: [['composerView', 'messageView', 'attachmentBoxViewOwner']],
+    identifyingFields: [['composerView', 'messageViewOwner', 'attachmentBoxViewOwner']],
```

Comment:
Note: to finish it this clean, we should also make the same rename to `composerViewOwner` in another PR.

Reply:
This doesn't create much diff, so I included it inside this PR.

Reply:
Well it does make the PR a bit harder to review than the 2 separate parts, but ok now that it's done.